CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
 
-## [v0.4.1](https://github.com/docker/notary/releases/tag/v0.4.0) 9/27/2016
+## v0.5-dev (in progress)
++ Non-certificate public keys in PEM format can now be added to delegation roles [#965](https://github.com/docker/notary/pull/965)
++ PostgresSQL support as a storage backend for Server and Signer [#920](https://github.com/docker/notary/pull/920)
++ Notary server's health check now fails if it cannot connect to the signer, since no new repositories can be created and existing repositories cannot be updated if the server cannot reach the signer [#952](https://github.com/docker/notary/pull/952)
++ Server runs its connectivity healthcheck to the server once every 10 seconds instead of once every minute. [#902](https://github.com/docker/notary/pull/902)
++ The keys on disk are now stored in the `~/.notary/private` directory, rather than in a key hierarchy that separates them by GUN and by role.  Notary will automatically migrate old-style directory layouts to the new style.  **This is not forwards-compatible against notary<0.4.1 and docker<=1.12** [#872](https://github.com/docker/notary/pull/872)
```

Comment:
Hmm... 0.4.1 does have the relevant change, and it is tagged.  But it is also not released.  I think 0.4.1 is technically more correct, but 0.4.2 is more friendly.  Will be happy to make the change if you feel it'd be better